integration-box/csv_import_via_http/csv_import_td.py

```diff
@@ -0,0 +1,20 @@
+import os
+import sys
+import pandas as pd
+import json
+
+os.system(f"{sys.executable} -m pip install -U pytd==1.0.0 td-client")
```

Comment:
`td-client` doesn't need to upgrade explicitly, pytd will introduce it as well.

Reply:
@chezou thx for advice! but when I tried to run this script with no specified 'td-client', workflow showed me error as following.
```
  File "/csv_import_td.py", line 26, in main
    client.load_table_from_dataframe(droped_df, table, writer='bulk_import', if_exists='append')
TypeError: load_table_from_dataframe() got an unexpected keyword argument 'writer'`
```

then, I added `td-client` and can fix it. do you have any idea?

Reply:
hmm, sounds odd. but ok for now to keep putting `td-client`. 

The error came from the pytd and it seems upgrading pytd has been failed. Technically, this should be a different cause whether including `td-client` or not.